cmd/jaeger/internal/extension/remotesampling/config.go

```diff
@@ -0,0 +1,103 @@
+	// all operations.
+	// TODO(wjang): Expand on why this is needed when BucketsForCalculation seems to suffice.
+	AggregationBuckets int `mapstructure:"aggregation_buckets"`
+
+	// BucketsForCalculation determines how many previous buckets used in calculating the weighted QPS,
+	// ie. if BucketsForCalculation is 1, only the most recent bucket will be used in calculating the weighted QPS.
+	BucketsForCalculation int `mapstructure:"buckets_for_calculation"`
+
+	// Delay is the amount of time to delay probability generation by, ie. if the CalculationInterval
+	// is 1 minute, the number of buckets is 10, and the delay is 2 minutes, then at one time
+	// we'll have [now()-12m,now()-2m] range of throughput data in memory to base the calculations
+	// off of. This delay is necessary to counteract the rate at which the jaeger clients poll for
+	// the latest sampling probabilities. The default client poll rate is 1 minute, which means that
+	// during any 1 minute interval, the clients will be fetching new probabilities in a uniformly
+	// distributed manner throughout the 1 minute window. By setting the delay to 2 minutes, we can
+	// guarantee that all clients can use the latest calculated probabilities for at least 1 minute.
+	Delay time.Duration `mapstructure:"delay"`
+
+	// InitialSamplingProbability is the initial sampling probability for all new operations.
+	InitialSamplingProbability float64 `mapstructure:"initial_sampling_probability"`
+
+	// MinSamplingProbability is the minimum sampling probability for all operations. ie. the calculated sampling
+	// probability will be in the range [MinSamplingProbability, 1.0].
+	MinSamplingProbability float64 `mapstructure:"min_sampling_probability"`
+
+	// MinSamplesPerSecond determines the min number of traces that are sampled per second.
+	// For example, if the value is 0.01666666666 (one every minute), then the sampling processor will do
+	// its best to sample at least one trace a minute for an operation. This is useful for low QPS operations
+	// that may never be sampled by the probabilistic sampler.
+	MinSamplesPerSecond float64 `mapstructure:"min_samples_per_second"`
+
+	// LeaderLeaseRefreshInterval is the duration to sleep if this processor is elected leader before
+	// attempting to renew the lease on the leader lock. NB. This should be less than FollowerLeaseRefreshInterval
+	// to reduce lock thrashing.
+	LeaderLeaseRefreshInterval time.Duration `mapstructure:"leader_lease_refresh_interval"`
+
+	// FollowerLeaseRefreshInterval is the duration to sleep if this processor is a follower
+	// (ie. failed to gain the leader lock).
+	FollowerLeaseRefreshInterval time.Duration `mapstructure:"follower_lease_refresh_interval"`
+}
+
+type Config struct {
+	File     FileConfig     `mapstructure:"file"`
+	Adaptive AdaptiveConfig `mapstructure:"adaptive"`
+	Port     string         `mapstructure:"port"`
```

Comment:
temporary? It needs to be full blown server config, e.g. `go.opentelemetry.io/collector/config/confighttp.ServerConfig`